scripts/e2e/delete_fixtures

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+# Does the inverse of create_fixtures
+echo && echo "[$0] deleting fixtures for api e2e tests..."
+
+delete_doc() {
+	local docUrl="$COUCH_URL/$1"
+	local rev="$(curl $docUrl | jq --raw-output ._rev)"
```

Comment:
Suggesting:

    local rev="$(curl "$docUrl" | jq --raw-output ._rev)"

Because it would be safer.  But I guess it's unlikely that doc URL would ever contain a space, so perhaps OK.